cdap-app-templates/cdap-etl/cdap-etl-batch-app/src/main/java/co/cask/cdap/app/etl/batch/config/ETLBatchConfig.java

```diff
@@ -30,6 +30,13 @@
   private final String schedule;
   private final List<ETLStage> actions;
 
+  public ETLBatchConfig(String schedule, ETLStage source, List<ETLStage> sinks, List<ETLStage> transforms,
+                        Resources resources, List<ETLStage> actions) {
+    super(source, sinks, transforms, resources);
```

Comment:
not related on this file, but i noticed in ``ETLConfig`` (super class)  we have a variable for  ETLStage for backwards compatibility along with List<ETLStage>, a comment on how long it will be there or a todo will be useful there.